docs/source/git-and-github/github-basics.rst

```diff
@@ -18,17 +18,13 @@ Creating a Repository
 Creating a repository is simple. We'll do this through the `GitHub Website <https://www.github.com>`_. First go to the `AguaClara Team Page <https://github.com/AguaClara>`_.
 
 
-.. image:: https://github.com/AguaClara/aguaclara_tutorial/wiki/Images/AguaClaraHome.png
-   :target: https://github.com/AguaClara/aguaclara_tutorial/wiki/Images/AguaClaraHome.png
-   :alt: AguaClaraHomePage
+.. image:: images/git11.PNG
```

Comment:
The preview on Github isn't showing this image, I think it needs to be `images/git11.png` (lowercase).